qiskit/compiler/assemble.py

```diff
@@ -297,6 +320,7 @@ def _parse_pulse_args(backend, qubit_lo_freq, meas_lo_freq, qubit_lo_range,
                            meas_map=meas_map,
                            memory_slot_size=memory_slot_size,
                            rep_time=rep_time,
+                           rep_delay=rep_delay,
```

Comment:
What if both are passed? Then the behavior is determined by `dynamic_reprate_enabled`, right? Should only `rep_time` be passed if dynamic_reprate_enabled is False, and only `rep_delay` passed if dynamic_reprate_enabled is True? Maybe it doesn't really matter

Reply:
Yes. I think it's fine to pass it--it won't influence the behavior...